test/__snapshots__/index.test.ts.snap

```diff
@@ -2006,6 +2714,50 @@ Object {
 }
 `;
 
+exports[`select with head:true 1`] = `
+Object {
+  "body": null,
+  "count": null,
+  "data": null,
+  "error": null,
+  "status": 200,
+  "statusText": "OK",
+}
+`;
+
+exports[`select with head:true, count:estimated 1`] = `
+Object {
+  "body": null,
+  "count": 510,
```

Comment:
count: estimated outputs 510 when the actual count is 4. Does anyone have any suggestion about what to do with this?

Reply:
I just read that we need to run `ANALYZE` on the table to update the estimate count in the plan. This seems like a separate issue, so I think we should just leave this 510 for right now and maybe deal with it in a separate pull request. What does everybody think? @steve-chavez  @kiwicopple  @soedirgo

Reply:
Oh, so `estimated` and `planned` just give estimates, so the number can fluctuate sometimes. I think it's fine to either just test for `exact` or just snapshot the `data`.

Reply:
On second thought, not much point in testing if it doesn't test the count. Let's keep these in for now and hope it behaves...

Reply:
Sounds good!

Reply:
These numbers will definitely change based on the number of tuples Postgres needs to clean up. Perhaps we can use a [Property Matcher](https://jestjs.io/docs/en/snapshot-testing#property-matchers) to check that it's returning a(ny) number?

Reply:
@kiwicopple Great idea. I just updated the test code to do so!